fdb-record-layer-core/src/main/java/com/apple/foundationdb/record/provider/foundationdb/IndexingBase.java

```diff
@@ -425,6 +425,12 @@ private static IndexBuildProto.IndexBuildIndexingStamp blocklessStampOf(IndexBui
                 .build();
     }
 
+    private CompletableFuture<Void> failedFutureVoid(Throwable ex) {
```

Comment:
Perhaps it would be worthwhile to put this in `MoreAsyncUtil` and make it generic (`CompletableFuture<T>`). 
There's definitely other code doing this. (honestly I was a bit surprised it wasn't supported in `CompletableFuture` or already existing in `MoreAsyncUtil`)